terraform/pipeline.tf

```diff
@@ -72,4 +72,39 @@ resource "aws_iam_policy" "c14-bandcamp-pipeline-policy" {
 resource "aws_iam_role_policy_attachment" "c14-bandcamp-pipeline-policy-attachment" {
   role       = aws_iam_role.c14-bandcamp-pipeline-lambda-role.name
   policy_arn = aws_iam_policy.c14-bandcamp-pipeline-policy.arn
+}
```

Comment:
I'm being picky and this may be unnecessary we could add descriptions to resources e.g. Description: Attaches the IAM policy to the Lambda role

Reply:
Does the comment above not cover this? Some resources (like this one) don't support an explicit 'description' tag

Reply:
Yes true